extensions/2.0/Khronos/KHR_materials_clearcoat/README.md

```diff
@@ -62,19 +62,22 @@ All implementations should use the same calculations for the BRDF inputs. Implem
 |**clearcoatRoughnessTexture**     | [`textureInfo`](/specification/2.0/README.md#reference-textureInfo)             | The clearcoat layer roughness texture. | No                   |
 |**clearcoatNormalTexture**        | [`normalTextureInfo`](/specification/2.0/README.md#reference-normaltextureinfo) | The clearcoat normal map texture.      | No                   |
   
-The clearcoat formula `f_clearcoat` is computed using the specular term from the glTF 2.0 Metallic-Roughness material, defined in [Appendix B](/specification/2.0/README.md#appendix-b-brdf-implementation).  Use F0 equal `0.04`, base color white `1.0, 1.0, 1.0`, metallic value `0.0`, and the clearcoat roughness value defined in this extension as follows:
+The clearcoat formula `f_clearcoat` is computed using the specular term from the glTF 2.0 Metallic-Roughness material, defined in [Appendix B](/specification/2.0/README.md#appendix-b-brdf-implementation).  Use specular F0 equal `0.04`, base color black `0.0, 0.0, 0.0`, metallic value `0.0`, and the clearcoat roughness value defined in this extension as follows:
 
 ```
 clearcoatRoughness = clearcoatRoughnessFactor * clearcoatRoughnessTexture.g
 ```
 
+The resulting `f_clearcoat` color is expected to have a Fresnel term blended into it as a result of the above.
```

Comment:
I am still confused about how `f_clearcoat` is now computed... Why not write it down explicitly? Something like:

> The clearcoat formula `f_clearcoat` is computed using the specular term from the glTF 2.0 Metallic-Roughness material, defined in [Appendix B](/specification/2.0/README.md#appendix-b-brdf-implementation).
>
> ```
> f_clearcoat = Fc * G * D / (4 * NdotL * NdotV)
> ```
>
> with Fresnel for dielectrics and index of refraction of 1.5
>
> ```
> Fc = 0.04 + 0.96 * (1 - VdotH)^5
> ```
>
> The roughness value used in the G and D terms is computed from the `clearcoatRoughness`: `alpha = clearcoatRoughness^2`.

Reply:
Doesn't this repeat formulas that are specified in Appendix B?  We already reference that appendix at least twice.

I'm not opposed to the idea, but I'm eager to move this proposal out of the "new ideas" phase and onto ratification.  There are several implementations already deployed.

Reply:
I don't have a strong opinion, I just thought that repeating the formula instead of text makes it more clear, avoids misunderstanding.

Reply:
I pulled out this sentence.  I had hoped to explain a long chain of emails with a quick note, but this clearly wasn't the correct way to do it.